requirements.txt

```diff
@@ -1,4 +1,3 @@
-git+https://github.com/mdellweg/python-dbus-next@master
 .
```

Comment:
```suggestion
.
dbus-next
```

Reply:
The dependency is already declared in setup.py.